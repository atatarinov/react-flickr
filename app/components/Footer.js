import React from 'react';

export default function Footer() {

  return (
    <div className="footer">
      <p>Built with<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/1024px-React-icon.svg.png" />by Alex Tatarinov</p>
    </div>
  );
}
