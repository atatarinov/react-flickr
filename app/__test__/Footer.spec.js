import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

test('Footer renders correnctly', () => {
  const component = renderer.create(<Footer />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
