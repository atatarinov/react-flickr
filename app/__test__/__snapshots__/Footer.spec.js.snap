// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correnctly 1`] = `
<div
  className="footer"
>
  <p>
    Built with
    <img
      src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/1024px-React-icon.svg.png"
    />
    by Alex Tatarinov
  </p>
</div>
`;
